frontend/BoardGUI.py

```diff
@@ -1,20 +1,29 @@
 import pygame as pg
-# from backend.Board import Board
+
+from backend.Board import Board
```

Comment:
I maintain that we should not be importing Board here (separation of frontend and backend; Board gets initialized but never used outside of tests), but that's not something that needs to be fixed for this sprint.

Reply:
Lets update this next spring. I agree it should be built from the GameManager I believe

Reply:
https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/32

Reply:
https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/33